AraEvent/AraEventCalibrator.cxx

```diff
@@ -1158,6 +1182,8 @@ void AraEventCalibrator::UnpackDAQFormatToElecChanFormat(UsefulAtriStationEvent
             voltMapIt=theEvent->fVolts.find(chanId);
 
             int samp=0;
+            const int blockIndex = (blockList->at(chanId).size() > 0)?
```

Comment:
Okay, so I kind of hate this ? notation, I think it obscures meaning. Can we change it?

Reply:
I disagree strongly here :) (I would prefer it on one line though...)

Reply:
Will put this onto one line, but will keep the `?` notation so that the variable can be `const` 🙂

Reply:
Hahaha, touche.